libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorage.cs

```diff
@@ -80,6 +80,22 @@ public CosmosDbStorage(CosmosDbStorageOptions cosmosDbStorageOptions)
             _client = new DocumentClient(cosmosDbStorageOptions.CosmosDBEndpoint, cosmosDbStorageOptions.AuthKey, connectionPolicy);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CosmosDbStorage"/> class.
+        /// using the provided CosmosDB credentials, database ID, and collection ID.
+        /// </summary>
+        /// <param name="cosmosDbStorageOptions">Cosmos DB storage configuration options.</param>
+        /// <param name="jsonSerializer">Mind that the TypeNameHandling, NullvalueHandling and ContractResolver may need special TLC: Recommended settings are as follows:
```

Comment:
Consider changing the words here a bit:

If passing in a custom JsonSerializer, we recommend the following settings: